tests/pip/reqs-included.txt

```diff
@@ -0,0 +1,3 @@
+-r temp1.txt
+--requirement=temp2.txt
+nltk==3.2.1
```

Comment:
I didn't tested whether a hash is right. I tested the original issue - to check that after changing included req hash of container changes, too. I used hash values because it was done everywhere, so thought that it was not a big deal. In last commit I changed it to be right: check it without hardcoding hashes.